src/data/camera.ts

```diff
@@ -38,22 +38,25 @@ export const computeMJPEGStreamUrl = (entity: CameraEntity) =>
 
 export const fetchThumbnailUrlWithCache = (
   hass: HomeAssistant,
-  entityId: string
+  entityId: string,
+  ...args: any[]
 ) =>
   timeCachePromiseFunc(
     "_cameraTmbUrl",
     9000,
     fetchThumbnailUrl,
     hass,
-    entityId
+    entityId,
+    args
```

Comment:
Spread them out again to mimic the original function

```suggestion
    ...args
```